openmdao/components/indep_var_comp.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, name, val=None, **kwargs):
                              "`str` or an iterable of tuples of the form (name, value) or "
                              "(name, value, keyword_dict).")
 
```

Comment:
May want to give IndepVarComp a sys_jacobian function that is just a `pass` so that it doesn't do anything.